cmd/thanos/downsample.go

```diff
@@ -97,25 +87,8 @@ func runDownsample(
 			cancel()
 		})
 	}
-	// Start metric and profiling endpoints.
```

Comment:
do we not want to add back in `metricHTTPListenGroup()` if we are removing this block?

Reply:
Thanks for sharp eye here! But no metricHTTPListenGroup is intentional here. This is only CLI command.